core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -780,6 +800,26 @@ public function getModules(Contao\DataContainer $dc)
 		return $arrModules;
 	}
 
+	/**
+	 * Return all content elements
+	 *
+	 * @return array
+	 */
+	public function getContentElements()
+	{
+		$groups = array();
+
+		foreach ($GLOBALS['TL_CTE'] as $k=>$v)
+		{
+			foreach (array_keys($v) as $kk)
+			{
+				$groups[$k][] = $kk;
+			}
+		}
```

Comment:
Wouldn't this code be the same and much simpler?

```php
foreach ($GLOBALS['TL_CTE'] as $k=>$v)
{
    $groups[$k] = array_keys($v);
}
```

or just 

```php
$groups = array_map('array_keys', $GLOBALS['TL_CTE']);
```

Reply:
Changed in 5e98690dbfdd866dbded99a7aed4e7bda73e04f5.